haskell-src/Concordium/Types/Execution.hs

```diff
@@ -1605,6 +1624,11 @@ instance AE.ToJSON Event where
         "delegatorId" .= edrDelegatorId,
         "account" .= edrAccount
       ]
+    Upgraded {..} -> AE.object [
+        "tag" .= AE.String "Upgraded",
```

Comment:
This does not match the `FromJson` instance.
The `euAddress` seems to be missing